Jenkinsfile_CNP

```diff
@@ -1,12 +1,10 @@
 #!groovy
 properties([
         parameters([
-                string(name: 'URL_TO_TEST', defaultValue: 'https://manage-case.aat.platform.hmcts.net/list/case?jurisdiction=sscs', description: 'The URL you want to run these tests against'),
+                string(name: 'URL_TO_TEST', defaultValue: '', description: 'The URL you want to run these tests against'),
+                string(name: 'TRIBUNALS_API_URI', defaultValue: '', description: 'The API service to create test data'),
                 string(name: 'CCD_GATEWAY_URL', defaultValue: 'https://gateway-ccd.aat.platform.hmcts.net', description: 'The CCD Gateway URL these tests will need to run'),
-                string(name: 'TRIBUNALS_API_URI', defaultValue: 'http://sscs-tribunals-api-aat.service.core-compute-aat.internal', description: 'The API service to create test data'),
-                string(name: 'NUM_TESTS_IN_PARALLEL', defaultValue: '50', description: 'The number of tests to run in parallel'),
```

Comment:
Any reason for the parallelization value was removed....?

Reply:
This was a legacy protractor config setting.This is not required for Playwright as it has its on playwright config file to configure this setting.